<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Env</class>
 <widget class="QWidget" name="Env">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Env
{
	background-image: url(:/new/pic/timg.jpg);
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>501</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,3,1">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(245, 121, 0);
font: 24pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Temperature:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_temp">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 248, 0);</string>
        </property>
        <property name="digitCount">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 25pt &quot;Ubuntu&quot;;
color: rgb(245, 121, 0);</string>
        </property>
        <property name="text">
         <string>â„ƒ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,3,1">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(245, 121, 0);
font: 24pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Humidity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_hum">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 248, 0);</string>
        </property>
        <property name="digitCount">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: rgb(245, 121, 0);
font: 25pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>%rh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,3,1">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;Ubuntu&quot;;
color: rgb(245, 121, 0);</string>
        </property>
        <property name="text">
         <string>Ammonia:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_amm">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 248, 0);</string>
        </property>
        <property name="digitCount">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 25pt &quot;Ubuntu&quot;;
color: rgb(245, 121, 0);</string>
        </property>
        <property name="text">
         <string>ppm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="lcd_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>781</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(39, 251, 255);</string>
   </property>
   <property name="digitCount">
    <number>26</number>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>170</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,0,3">
    <item>
     <widget class="QPushButton" name="btn_th">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(14,23,52);
color: rgb(245, 121, 0);</string>
      </property>
      <property name="text">
       <string>Set threshold</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btn_back">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(14,23,52);
color: rgb(245, 121, 0);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
